File: app/src/main/java/com/partsystem/partvisitapp/core/database/entity/InvoiceCategoryCenterEntity.java
package com.partsystem.partvisitapp.core.database.entity;

import androidx.room.Entity;

import java.io.Serializable;

@Entity(tableName = "invoice_category_center_table")
public class InvoiceCategoryCenterEntity implements Serializable {
    public Integer InvoiceCategoryId;

    public Integer CenterId;
}

File: app/src/main/java/com/partsystem/partvisitapp/core/database/entity/InvoiceCategoryCenterEntity.kt
package com.partsystem.partvisitapp.core.database.entity

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "InvoiceCategoryCenter")
data class InvoiceCategoryCenterEntity(
    @PrimaryKey val InvoiceCategoryId: Int,
    val CenterId: Int,
)
